### PR TITLE
fix(nav): rsweb-7138 Nav Fix

### PR DESCRIPTION
**Pull request by SyntaxStacks**
https://jira.rax.io/browse/RSWEB-7138

### CODE REVIEW

**Comment by runrog** [comment]
👍  works good

File: styleguide/js/zoolander.js [hunk 1]
```diff
@@ -44,6 +44,7 @@ jQuery(document).ready(function($) {
         .on('touchend', dropDownTrigger);
   } else {
     $(".navbar-tertiary-dropDownTrigger").hover(dropDownTrigger);
+    $('.navbar-tertiary-dropDownTrigger').on('click', function (e) { e.stopPropagation(); });
```
Comment (WidgetsBurritos) [inline styleguide/js/zoolander.js]: Should this be combined with the previous selector look up? 

```
    $(".navbar-tertiary-dropDownTrigger").hover(dropDownTrigger).on('click', function (e) {
      e.stopPropagation();
    });
```

**Comment by WidgetsBurritos** [comment]
👍

**Comment by xinbin** [comment]
This has a thumbs up and I'm merging it in so the nav works as expected in zoolander 💃

**Pull request merged**